### PR TITLE
Carson Jardine IC 1 Pull Request

### PR DESCRIPTION
**Pull request by carson-jardine**
Added additional code to convert end time to clock time for schools that end before 1pm and do not need to be converted

### CODE REVIEW

**Comment by KathleenScriver** [comment]
Great job with your commits.

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  def end_time
+    (@start_time.to_i + @hours_in_school_day).to_s + ':00'
+  end
+
+  def is_full_time?
+    @hours_in_school_day > 4
+  end
+
+  def standard_student_names
+    @student_names.map do |name|
```
Comment (KathleenScriver) [inline lib/school.rb:23]: Great job with the #map here.

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(student)
+    @student_names << student
+  end
+
+  def end_time
+    (@start_time.to_i + @hours_in_school_day).to_s + ':00'
```
Comment (KathleenScriver) [inline lib/school.rb:15]: Great job recognizing that you could just call to_i on the string for this one.

File: lib/school.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  end
+
+  def is_full_time?
+    @hours_in_school_day > 4
```
Comment (KathleenScriver) [inline lib/school.rb:19]: Perfect!

File: lib/school.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  end
+
+  def standard_student_names
+    @student_names.map do |name|
+      name.capitalize
+    end
+  end
+
+  def convert_end_time_to_clock_time
+    if self.end_time.to_i >= 13
```
Comment (KathleenScriver) [inline lib/school.rb:29]: self isn't needed here, you can just do `@end_time.to_i` and that should work just fine.

File: test/school_test.rb [hunk 5]
```diff
@@ -45,11 +45,37 @@ def test_it_can_add_student_names
...
 
     assert_equal '16:00', school1.end_time
     assert_equal '12:00', school2.end_time
   end
+
+  #Iteration 3 Tests:
+  def test_it_is_full_time
+    school = School.new('9:00', 7)
+
+    assert school.is_full_time?
```
Comment (KathleenScriver) [inline test/school_test.rb:60]: For a method that can return a few different return values, in this case either true or false, I would want to see that the test tests both of those values. So would want a second assert_equal false, school.is_full_time? here.

File: lib/school.rb (hunk 4 above)
Comment (carson-jardine) [reply lib/school.rb:29]: good to know, I need to work on my understanding of self :)

File: test/school_test.rb (hunk 5 above)
Comment (carson-jardine) [reply test/school_test.rb:60]: Makes sense, I'll keep that in mind when writing future tests!

**Pull request closed without merging**